components/edit-collective/EditCollectiveArchive.js

```diff
@@ -153,17 +153,20 @@ const ArchiveCollective = ({ collective, archiveCollective, unarchiveCollective
       <Modal show={modal.show} width="570px" onClose={() => setModal({ ...modal, show: false })}>
         <ModalHeader>
           <FormattedMessage
-            values={{ type: collectiveType.toLowerCase() }}
+            values={{
+              type: collectiveType.toLowerCase(),
+              actionType: modal.type,
+            }}
             id="collective.archive.modal.header"
-            defaultMessage={'Archive {type}'}
+            defaultMessage={'{actionType} {type}'}
```

Comment:
@Betree what would be the best practice for this kind of strings regarding internationalization?

Reply:
We should not do that because this has strictly no effect, the string can't be translated.

A potential solution is to do something this:

```es6
const archiveMessages = defineMessages({
  [type.EVENT]: {
    id: 'Archive.Event',
    defaultMessage: 'Archive this event'
  },
  _default: {
    id: 'Archive.Account',
    defaultMessage: 'Archive this account'
  }
});

const { formatMessage } = useIntl();

return (
  <div>
    {formatMessage(archiveMessages[collectiveType] || archiveMessages._default)}
  </div>
)
```

Reply:
Or another solution from https://docs.opencollective.com/help/contributing/development/translations:

```jsx
<FormattedMessage
    id="ArchiveAccount"
    defaultMessage="Archive {type, select, EVENT {this event} COLLECTIVE {this collective} other {this account}}"
    values={{ type: collective.type }}
/>
```

Reply:
@flickz for "Unarchive" vs "Archive", the best is to have different strings for the messages, instead of trying to use the same message. For the type, @Betree's suggestion looks really great.